### ISSUE TITLE
parseError: Spread must be the final element of an element list

### ISSUE THREAD

**Issue by dtinth** [jscs-dev/node-jscs#862]
This happens when creating an array with a spread:

```js
      let buffers = [
            new Buffer('BEMUSEPACK'),
            size,
            metadata,
            ...this._buffers,
          ]
```

This is perfectly valid JavaScript. The trailing comma after `...this._buffers` is there because we `requireTrailingComma`.

**Comment by markelog** [comment]
I'm afraid until <a href="https://github.com/ariya/esprima/tree/harmony">esprima</a> would not have support for it, we can't do anything about it.

Sorry.

**Comment by okonet** [comment]
Can it at least be disabled by a comment?

**Comment by hzoo** [comment]
You can use babel-jacks now as the esprima parser. https://github.com/jscs-dev/babel-jscs/releases

**Comment by okonet** [comment]
Thanks but until this https://github.com/jscs-dev/babel-jscs/issues/1 is open, it's unusable for my setup.

**Comment by hzoo** [comment]
Ok fixed jscs-dev/babel-jscs#1 in 1.0.3

**Comment by okonet** [comment]
🎉👏
